sigv4.go

```diff
@@ -0,0 +1,105 @@
+package appsync
+
+import (
+	"bytes"
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"errors"
+	"log"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	sdkv2_v4 "github.com/aws/aws-sdk-go-v2/aws/signer/v4"
+	sdkv1_v4 "github.com/aws/aws-sdk-go/aws/signer/v4"
+)
+
+type sigv4 interface {
+	signHTTP(payload []byte) (http.Header, error)
```

Comment:
It's better to remove argument names or use the same name as the methods

Reply:
Sorry, I don't get your intent of "use the same name as the method".

For example, cloud
```
signHTTP(http []byte) (http.Header, error)
```
meet your expectations?

Reply:
I'd just like to ask difference in use between `body []byte` of the implemented method and `payload []byte` of interface.

Reply:
Thanks, I understand.
I will update it soon.